script.js

```diff
@@ -1,5 +1,203 @@
-function runProgram(){
+function displayPresInformation(president){
+    console.log(president);
 
+    let CardContainer = document.querySelector(".bottom-container")
+    let newCard = createCard(president);
+    CardContainer.appendChild(newCard);
+
+}
+
+function displayPresName(president){
+    console.log("function displayPresName running");
+    let NameContainer = document.querySelector(".bottom-container");
+    let NameCard = createNameCard(president);
+    NameContainer.appendChild(NameCard);
+}
+
+function createNameCard(president){
+    let nameCard = document.createElement('p');
+    nameCard.classList.add('.info-container');
+
+    let PresName = document.querySelector('div');
+    PresName.classList.add('card-body');
+
+    let name = document.createElement('p');
+    name.textContent = `Name: ${president.name}`;
+    name.classList('card-title');
+    
+    PresName.appendChild(name);
+    nameCard.appendChild(nameCard);
+    
+    return PresCard;
```

Comment:
Then instead of this returning function you have, do the following:

- [ ] get the search input
- [ ] see if the search input matches each president
- [ ] display each of those presidents with whatever filter value happens to be set:


``` js
//get search input
    let searchInput = document.querySelector("#search-input");
    console.log(searchInput);
    //clear the container
    clearContainer();

    //filter the presidents by name
    let nameMatches = presidents.filter(president => president.name.toLowerCase().includes(searchInput.value.toLowerCase()));

    //get the filter value
    let filterValue = document.querySelector("#presidentFilter").value;

    //display the filtered presidents
    for(let president of nameMatches){
        displayPresInformation(president, filterValue);
    }
```